lightning/src/util/persist.rs

```diff
@@ -134,9 +136,9 @@ pub trait KVStore {
 	///
 	/// Will create the given `primary_namespace` and `secondary_namespace` if not already present
 	/// in the store.
-	fn write(
+	fn write_async(
```

Comment:
Reminder: document that these calls, even though they are async, need to complete in the order in which they are initiated.